src/bson/bson.c

```diff
@@ -2681,7 +2681,7 @@ _bson_as_json_visit_bool (const bson_iter_t *iter,
 {
    bson_json_state_t *state = data;
 
-   bson_string_append (state->str, v_bool ? "true" : "false");
+   bson_string_append (state->str, v_bool ? "true":"false");
```

Comment:
Correct. Seems I removed one to many spaces..
Would be great if the spaces were optional though. I assume most people using this method use it (like me) to generate a json string to send back to a client. Adding all this spaces for no reason makes the response larger without adding any functionality.